doc/api_assets/style.css

```diff
@@ -741,6 +728,28 @@ kbd {
   outline: var(--brand3) dotted 2px;
 }
 
+@media only screen and (max-width: 576px) {
+  #gtoc > ul > li {
+    display: block;
+    border-right: none;
+  }
+}
+
```

Comment:
This block doesn't seem to be necessary anymore.
```suggestion
```

Reply:
ack, removed it. Thanks!